packages/driver/src/cypress/error_utils.js

```diff
@@ -431,7 +440,40 @@ const errorFromProjectRejectionEvent = (event) => {
   }
 
   // makeErrFromObj clones the error, so the original doesn't get mutated
-  return [makeErrFromObj(event.reason), event.promise]
+  return {
+    originalErr: event.reason,
+    err: makeErrFromObj(event.reason),
+    promise: event.promise,
+  }
+}
+
+const errorFromUncaughtEvent = (handlerType, event) => {
+  return handlerType === 'error' ?
+    errorFromErrorEvent(event) :
+    errorFromProjectRejectionEvent(event)
+}
+
+const logError = (Cypress, handlerType, err, handled = false) => {
+  Cypress.log({
+    message: err.message,
```

Comment:
Any reason for not using err.name too?

```suggestion
    message: `${err.name}: ${err.message}`,
```

Reply:
No reason, really. I don't usually find the name to be very useful and there's limited real estate to display the message, but I'm okay with adding it.